### ISSUE TITLE
The argument "URL" is not recognised.

### ISSUE THREAD

**Issue by chevdor** [afonsoc12/firefly-cli#17]
```
🐷 ➜ edit URL https://firefly.my.domain.com

The argument "URL" is not recognised.
```

**Comment by chevdor** [comment]
I ran into this issue in both interactive and calling from the command line with:
```
ff edit URL https://firefly.my.domain.com
The argument "URL" is not recognised.
```

**Comment by chevdor** [comment]
I saw that the ini file was created but empty.
Editing the file manually did solve my problem but the issue remains I think.

**Comment by afonsoc12** [comment]
Hi,
Thanks for reporting this. Will be fixed on v0.2.2!
Wait 15 mins and update your release
